### DISCUSSION TITLE
Debian/MariaDB

### DISCUSSION THREAD

**Discussion by uniquwy** [eusonlito/GPS-Tracker#83] [Q&A]
Hi!

For the reason that Debian distro was totally switched from the mainline MySQL to scanty fork named MariaDB, your app generally cannot be installed on the Debian systems without wide system remake (especially if system already works in the production with a stock packages).

There is patch, which allows to adapt current version ([6ca836d4262de3b88ef5b2e790f6eb300bc5cebd](https://github.com/eusonlito/GPS-Tracker/tree/6ca836d4262de3b88ef5b2e790f6eb300bc5cebd)) of app for the Debian 11+, but due to mariadb functionality lack, many solutions in the patch will certainly seem dubious and non-elegant. I don't know what to do with this next, my own issue is solved, but I want to share my work done.

[mariadb.patch](https://github.com/user-attachments/files/17788312/mariadb.patch)
(place file to the project root and run: _**git apply mariadb.patch**_)

**Comment by eusonlito** [comment]
MariaDB is not supported: https://github.com/eusonlito/GPS-Tracker/issues/18#issuecomment-1621767898

**Reply by uniquwy** [reply]
But it works for me now :)

**Reply by eusonlito** [reply]
Great!
